btrack/_localization.py

```diff
@@ -70,19 +64,47 @@ def _is_multiclass(x: np.ndarray) -> bool:
     if CENTROID_PROPERTY not in properties:
         properties = (CENTROID_PROPERTY,) + properties
 
-    # check to see whether this is a binary segmentation or multiclass
-    if _is_binary(segmentation):
-        labeled = label(segmentation)
-    elif _is_ternary(segmentation):
-        labeled = label(segmentation)
-    elif _is_multiclass(segmentation):
+    # if class id is specified then extract that property first
+    if assign_class_ID:
+        # check to see if class ID can be read from segmentation
+        # there are a few cases where this wouldnt work ie if there is a single
+        # instance of any one cell type, maybe best to just skip this check
+        # if _is_unique(segmentation):
```

Comment:
and i think i ought to just get rid of this check as there are a few instances where it would stop the class id labels from being processed when they should be